doc/classes/TabContainer.xml

```diff
@@ -187,36 +187,52 @@
 	</constants>
 	<theme_items>
 		<theme_item name="decrement" type="Texture2D">
+			Icon for the left arrow that appears when there's too many tabs to fit them, when it's impossible to click it.
```

Comment:
```suggestion
			Icon for the left arrow button that appears when there are too many tabs to fit in the container width. Used when the button is disabled (i.e. the first tab is selected).
```

Reply:
The `decrement` / `decrement_highlight` are pretty confusing names BTW given what they are actually used for.

But that's out of scope of this PR. Maybe we should have a tracker to review all `theme_item`s to see both whether they are used, relevant and well named?